packages/babel-preset-babili/src/index.js

```diff
@@ -1,20 +1,50 @@
+const err = new Error("Babili requires babel-core>=6.14.0. Upgrade babel-core or tools dependent on babel-core to recent versions");
+
+function isVersion(versionStr) {
+  const version = versionStr.split(".").map((p) => parseInt(p));
+  return version[0] >= 6 && version[1] >= 14;
```

Comment:
It does show up properly. But problems in supporting older versions.

+ Older versions of babel only accept an object
+ Newer versions when they accept an object with `key = plugins` don't execute `buildPreset` - so no plugin is applied in preset.
+ We use `name` property to measure performance of each plugin.